app/controllers/users.js

```diff
@@ -54,14 +54,18 @@ module.exports = {
         }
         if (bcrypt.compareSync(password, user.password)) {
           logger.info(`User ${email} authenticated.`);
-          const token = jwt.sign(JSON.stringify(user), config.secret);
-          return res.status(200).json({ token });
+          const token = jwt.sign(JSON.parse(JSON.stringify(user)), config.secret, {
+            expiresIn: config.expirationInSeconds
```

Comment:
the package automatically uses this attribute to check? How does it knows when it was created the token?

Usually in normal jwt you can use the attribute `exp` with the expirationDate an it will be checked automatically.

Reply:
Yes, it actually adds that amount of time to Date.now()